### ISSUE TITLE
Use another key instead of SUPER for tiling

### ISSUE THREAD

**Issue by 01mf02** [pop-os/cosmic-settings#522]
In my current tiling WM, I use the ALT key for all tiling-based behaviour, e.g. ALT-J / ALT-K for cycling through windows, ALT-Space for toggling fullscreen and so on. I do this because I found using the SUPER key to cause hand pain in the long term. It would be great if it was possible to configure another key, such as ALT, instead of SUPER for tiling operations.

**Comment by rxt30** [comment]
It's already possible by going to "Input Devices"->Keyboard->"View and customize shortcuts" and adding shortcuts with the ALT key.

**Comment by kammerchorinnsbruck** [comment]
@rxt30, yes, but it means modifying each and every shortcut one by one. It is possible, certainly, but it could be more efficient.

**Comment by 01mf02** [comment]
@rxt30, yes, but it means modifying each and every shortcut one by one. It is possible, certainly, but it could be more efficient.
